### DISCUSSION TITLE
Remove unnecessary hierarchy in the basic tutorial.

### DISCUSSION THREAD

**Discussion by jinalskothari** [kubernetes/website#43596] [Ideas]
For example `Create a Cluster`. The top-level page simply has a link to the child `Using Minikube to Create a Cluster`

**Comment by divya-mohan0209** [comment]
Hello @jinalskothari , please [open an issue](https://github.com/kubernetes/website/issues/new?assignees=&labels=kind%2Ffeature&projects=&template=feature-request.md) against the kubernetes/website repository instead of starting up discussion items with your feature requests/suggestions.
